pkg/gadgettracermanager/api/gadgettracermanager.proto

```diff
@@ -68,6 +68,7 @@ message ContainerDefinition {
   string podname = 5;
   string name = 6;
   repeated Label labels = 7;
+  bool labels_set = 8;
```

Comment:
Maybe add a comment what it is for and that some hooks know the labels, some don't.